firmware/TempoHandler.h

```diff
@@ -49,38 +50,33 @@ class TempoHandler
     inline void setHandleAlignEvent(void (*fptr)()) {
       tAlignCallback = fptr;
     }
-    void setPPQN(int ppqn) {
-      _ppqn = ppqn;
-    }
```

Comment:
We never set PPQN to anything else than `24`, and currently things would break for the sequencer if we would, so it's better to keep this as a static value which is currently owned by `Sequencer`.